### PR TITLE
debug has log too much information.debug shodld log in retry.

### PR DESCRIPTION
**Pull request by hanrea**

### CODE REVIEW

**Comment by hanrea** [comment]
See ：https://github.com/sequelize/sequelize/issues/7281

File: index.js [hunk 1]
```diff
@@ -70,7 +69,7 @@ module.exports = function retryAsPromised(callback, options) {
 
       // Do some accounting
       options.$current++;
-
+      debug('Rerying '+ options.name + ' (%s)', options.$current);
```
Comment (eseliger) [inline index.js]: typo

**Comment by esetnik** [comment]
@hanrea can you fix this typo?

**Comment by mickhansen** [comment]
v2.3.1

**Comment by esetnik** [comment]
Thanks @hanrea and @mickhansen 👌

**Pull request merged**